bin/node/runtime/src/lib.rs

```diff
@@ -533,7 +533,7 @@ impl pallet_elections_phragmen::Trait for Runtime {
 	// NOTE: this implies that council's genesis members cannot be set directly and must come from
 	// this module.
 	type InitializeMembers = Council;
-	type CurrencyToVote = CurrencyToVoteHandler;
+	type CurrencyToVote = frame_support::traits::U128CurrencyToVote;
```

Comment:
Seems that all other traits are imported instead of using full path, I'd change that for consistency here as well.